backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5677570_sys_gh_doc_seq_country_based.sql

```diff
@@ -0,0 +1,429 @@
+-- Table: AD_DocType_CountryBased
+-- 2023-02-20T05:37:08.842Z
+INSERT INTO AD_Table (AccessLevel,ACTriggerLength,AD_Client_ID,AD_Org_ID,AD_Table_ID,CopyColumnsFromTable,Created,CreatedBy,Description,EntityType,ImportTable,IsActive,IsAutocomplete,IsChangeLog,IsDeleteable,IsDLM,IsEnableRemoteCacheInvalidation,IsHighVolume,IsSecurityEnabled,IsView,LoadSeq,Name,PersonalDataCategory,ReplicationType,TableName,TechnicalNote,TooltipType,Updated,UpdatedBy,WEBUI_View_PageLength) VALUES ('3',0,0,0,542310,'N',TO_TIMESTAMP('2023-02-20 06:37:08','YYYY-MM-DD HH24:MI:SS'),100,'Table is used for countrycodes in sequences based on billto countries','D','N','Y','N','N','Y','N','N','N','N','N',0,'DocType CountryBased','NP','L','AD_DocType_CountryBased','Table is used for countrycodes in sequences based on billto countries','DTI',TO_TIMESTAMP('2023-02-20 06:37:08','YYYY-MM-DD HH24:MI:SS'),100,0)
```

Comment:
i didn't check this SQL line-by-line, but it seems to lack documentation for the users.
- e.g. i as user need to know that currently it only works for invoice doctypes.
- this could be done via a tab description (afair we talked about this), and also by an MI that throws a userfriendly error if a  `AD_DocType_CountryBased` is created than there is no Provider in charge of C_Invoice